tests/testthat/setup.R

```diff
@@ -23,8 +23,7 @@ set.seed(101)
 
 shape <- 2
 rate <- 3
-
-# The mean of the gamma distribution is: shape * rate
+mu <- shape / rate
```

Comment:
and brms gamma is parameterised as the mean and the shape? not the shape and rate?

Reply:
These numbers are fed into `rgamma` in order to simulate data, which takes `shape` and `rate` input

Reply:
no I'm looking at mu here

Reply:
`mu` is used to compare the chains of `mu` parameters from the NUTS?